main/src/ca/uwaterloo/flix/api/lsp/provider/CodeActionProvider.scala

```diff
@@ -325,16 +325,18 @@ object CodeActionProvider {
     *   import java.io.File
     * }}}
     */
-  private def mkImportJava(name: String, uri: String, ap: AnchorPosition)(implicit root: Root): List[CodeAction] =
-    root.availableClasses.byClass.get(name).toList.flatten.map { path =>
-      val completePath = path.mkString(".") + "." + name
+  private def mkImportJava(name: String, namespace: List[Name.Ident], uri: String, ap: AnchorPosition)(implicit root: Root): List[CodeAction] = {
```

Comment:
I think we should update both the doc comment above and add a code comment below to explain what is happening.